storage/storage.cpp

```diff
@@ -616,7 +616,7 @@ void Storage::LoadCountriesFile(string const & pathToCountriesFile, string const
 
 int Storage::Subscribe(TChangeCountryFunction const & change, TProgressFunction const & progress)
 {
-  ASSERT_THREAD_CHECKER(m_threadChecker, ());
+//  ASSERT_THREAD_CHECKER(m_threadChecker, ());
```

Comment:
Экспериментировал, осталось случайно. Фиксед.
> On Apr 2, 2016, at 4:47 AM, Viktor Govako <notifications@github.com> wrote:
> 
> In storage/storage.cpp <https://github.com/mapsme/omim/pull/2696#discussion_r58286354>:
> 
> > @@ -616,7 +616,7 @@ void Storage::LoadCountriesFile(string const & pathToCountriesFile, string const
> >  
> >  int Storage::Subscribe(TChangeCountryFunction const & change, TProgressFunction const & progress)
> >  {
> > -  ASSERT_THREAD_CHECKER(m_threadChecker, ());
> > +//  ASSERT_THREAD_CHECKER(m_threadChecker, ());
> Что тут не так?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2696/files/a51fda6b0f744e8a5d59125843c2ab449a998754#r58286354>